docs/index.md

```diff
@@ -17,4 +17,7 @@ Once installed, whenever you run your app in development and view it in your bro
 - Keep recordings as short as possible -- focus on the one thing you want to understand better.
 - When you finish recording, CodeSee will open a new tab with your recording in it as soon as all the data has been transmitted. For larger recordings or on slower networks, there can be some buffering, so please give it a few seconds or more to finish sending the recording to our servers.
     - If it delays for long enough, your pop-up blocker may decide to block it. Keep an eye out for that if the codesee site isn't coming up. If in doubt, you can always go to [app.codesee.io/library](https://app.codesee.io/library) to see all your latest recordings.
-- The CodeSee UI can be a little sluggish with larger recordings! Sorry about that! For the moment, we ask that you give our UI a second to process now and then. But fear not, we take performance very seriously and we're actively working on performance improvements.
\ No newline at end of file
+- The CodeSee UI can be a little sluggish with larger recordings! Sorry about that! For the moment, we ask that you give our UI a second to process now and then. But fear not, we take performance very seriously and we're actively working on performance improvements.
+
+## [Installing CodeSee](../installation)
+&nbsp;  
```

Comment:
To me, this looks a little like we forgot to fill in the body of this header. Is this meant to be, like, a "Next step: Installing CodeSee" link?

Reply:
Sgtm